keps/2349-multikueue-external-custom-job-support/kep.yaml

```diff
@@ -0,0 +1,29 @@
+title: MultiKueue external custom Job support
+kep-number: 2349
+authors:
+  - "@khrm"
+status: implementable
+creation-date: 2025-07-15
+reviewers:
+  - "@tenzen-y"
+  - "@mimowo"
+  - "@trasc"
```

Comment:
I don't think this person is reviewing Kueue work.

Reply:
Sure. I would add you.